src/pke/lib/scheme/ckksrns/ckksrns-cryptoparameters.cpp

```diff
@@ -154,4 +159,30 @@ uint64_t CryptoParametersCKKSRNS::FindAuxPrimeStep() const {
     return static_cast<uint64_t>(2 * n);
 }
 
+void CryptoParametersCKKSRNS::ConfigureCompositeDegree(uint32_t scalingModSize) {
+    // Add logic to determine whether composite scaling is feasible or not
+    if (GetScalingTechnique() == COMPOSITESCALINGAUTO) {
+        if (NATIVEINT != 64)
+            OPENFHE_THROW(config_error, "COMPOSITESCALINGAUTO scaling technique only supported with NATIVEINT==64.");
+        uint32_t compositeDegree  = GetCompositeDegree();
```

Comment:
the line# !67
uint32_t compositeDegree  = GetCompositeDegree();
is not needed. compositeDegree should be defined on line# 171:
uint64_t compositeDegree = static_cast<uint32_t>(std::ceil(static_cast<float>(scalingModSize) / registerWordSize));

Reply:
Fixed in latest commit [c8c76a8](https://github.com/openfheorg/openfhe-development/pull/928/commits/c8c76a83e3b7fb232b5bf83f50218e4a3d1d920f).